### DISCUSSION TITLE
[Bug] single point of failure in chia.net introducers

### DISCUSSION THREAD

**Discussion by iimmann** [Chia-Network/chia-blockchain#13537] [Support]
### What happened?

Hi 
Yesterday something happened and I figure out a big bug in the network, if i understand correctly.
Im living in Iran, our stupid government has filtered many domains due to massive demonstrations. One of these domains was CHIA.NET
After that my full node couldn't sync anymore and my farm lost.
After a little investigation, I realized that all of the full nodes in the network are dependent on dns-introducer.chia.net to find peers on network so because of chia.net filtering we can't farm anymore!
This question arose in my mind, if a hacker takes the Chia domain out of reach, the entire network will be disrupted?!! only by hacking one domain?!!

### Version

1.6

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by shaneo257** [comment]
just checked https://dashboard.chia.net/d/em15uQ47k/peer-info?orgId=1 and peers from iran are still showing, are you sure its not an error with your setup

**Comment by emlowe** [comment]
The DNS introducer and the old introducer are hosted by chia.net. Although others could host additional introducers I don't believe there are any others at this time. This can make it difficult to bootstrap peers if those are blocked, however, one technique is to manually add some known peers - you could try to find some lists of peers and manually add them - maybe looking here would help https://www.reddit.com/r/chia/comments/n0eiqq/well_known_node_ips/

We would like there to be multiple introducers and we do expect there to be multiple running DNS introducers eventually
